src/itinerary/Itinerary.story.mdx

```diff
@@ -0,0 +1,331 @@
+    mainLabel: 'Porte de Vincennes',
+    subLabel: 'Paris',
+    actionAriaLabel: '09:00 Paris Porte de Vincennes (New page with a map)',
+    href: '#',
+  },
+  {
+    time: '12:00',
+    isoDate: '2017-12-11T12:00',
+    stepAriaLabel: 'Pick up/drop off location',
+    mainLabel: 'Gare de Tours',
+    subLabel: 'Tours',
+    actionAriaLabel: '09:00 Paris Porte de Vincennes (New page with a map)',
+    href: '#',
+  },
+  {
+    time: '15:00',
+    isoDate: '2017-12-11T15:00',
+    stepAriaLabel: 'Pick up/drop off location',
+    mainLabel: 'Gare',
+    subLabel: 'Nogent-le-Rotrou',
+    actionAriaLabel: '09:00 Paris Porte de Vincennes (New page with a map)',
+    href: '#',
+  },
+  {
+    time: '19:00',
+    isoDate: '2017-12-11T19:00',
+    stepAriaLabel: 'Pick up/drop off location',
+    mainLabel: 'Gare Bordeaux Saint-Jean',
+    subLabel: 'Bordeaux',
+  },
+]
+
+# **Itinerary**
+
+## Default
+
+<Preview>
+  <Story name="Default">
+    <BaseSection>
+      <h1 id="titleId">Title example</h1>
+      <Itinerary
+        aria-labelledby="titleId"
+        fromAddon="Lille"
+        fromAddonAriaLabel="Driver departure"
+        toAddon="Biarritz"
+        toAddonAriaLabel="Driver arrival"
+        places={placesWithOneStopover}
+        headline="Mon 11 December"
+        highlightRoad
+        collapsedLabel={`${placesWithStopovers.length} stops`}
+        collapsedAriaProps={{
+          'aria-label': 'Show or hide all stops',
+        }}
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With proximity
+
+<Preview>
+  <Story name="With proximity">
+    <BaseSection>
+      <Itinerary
+        fromAddon="Lille"
+        fromAddonAriaLabel="Driver departure"
+        toAddon="Biarritz"
+        toAddonAriaLabel="Driver arrival"
+        places={[
+          {
+            stepAriaLabel: 'Pick up location',
+            mainLabel: 'Paris',
+            subLabel: <Proximity value={Distances.FAR} title="Distance from the pick up point" />,
+            isoDate: '2017-12-11T09:00',
+            time: '09:00',
+            href: '#',
+          },
+          {
+            stepAriaLabel: 'Drop off location',
+            mainLabel: 'Bordeaux',
+            subLabel: (
+              <Proximity value={Distances.CLOSE} title="Distance from the drop off point" />
+            ),
+            isoDate: '2017-12-11T12:00',
+            time: '12:00',
+          },
+        ]}
+        headline="Mon 11 December"
+        highlightRoad
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With stopovers
+
+<Preview>
+  <Story name="With stopovers">
+    <BaseSection>
+      <Itinerary
+        aria-label="Ride plan"
+        fromAddon="Lille"
+        fromAddonAriaLabel="Driver departure"
+        toAddon="Biarritz"
+        toAddonAriaLabel="Driver arrival"
+        places={placesWithStopovers}
+        headline="Mon 11 December"
+        highlightRoad
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With segments & stopovers
```

Comment:
You could describe what is a segment, what's the expected data structure and how it is used.

Reply:
Done